common/pref_names.cc

```diff
@@ -16,6 +16,8 @@ const char kLastCheckYMD[] = "brave.stats.last_check_ymd";
 const char kLastCheckWOY[] = "brave.stats.last_check_woy";
 const char kLastCheckMonth[] = "brave.stats.last_check_month";
 const char kFirstCheckMade[] = "brave.stats.first_check_made";
+const char kThresholdCheckMade[] = "brave.stats.threshold_check_made";
```

Comment:
`chrome/common/pref_names.cc` has a good tradition of documenting prefs - can we put some explanations here as well?